src/index.js

```diff
@@ -260,6 +260,7 @@ function replaceName(filename, name) {
 }
 
 function walk(exports) {
+	if (!exports) return null;
 	if (typeof exports === 'string') return exports;
 	return walk(exports['.'] || exports.import || exports.module);
```

Comment:
Hmm... I found a regression when declaring modern exports using default conditional export, such as:

```json
"exports": {
    ".": {
      "import": "dist/index.js",
      "require": "dist/index.cjs",
      "default": "dist/index.modern.js"
    }
  }
```

The modern module is not being generated in version >= 0.13.2 while it is being generated in v0.13.1. Probably, we can also add `exports.default` inside the `walk` recursion method. wdyt @developit ? :bow: 

```suggestion
	return walk(exports['.'] || exports.import || exports.module || exports.default);
```

Reply:
any update regarding this fix? :bow:

Reply:
Hi @jeffryang24 - no update, just haven't gotten around to it yet.

Reply:
I think we could add `.default` as you suggest, but really this needs to also be checking the extension based on `pkg.type`, otherwise it will potentially return a CommonJS entry filename for the modern filename.

Reply:
I see... got it... Currently, I've reverted back to 0.13.1 to fix this regression issue. Thanks for your explanation... :bow:

Reply:
@jeffryang24 thanks, I also reverted to 0.13.1 and that makes it work for me for the moment.